packages/web/src/scss/components/Container/_theme.scss

```diff
@@ -1,6 +1,14 @@
 @use 'sass:map';
 @use '@tokens' as tokens;
 
-$container-max-width: tokens.$container-max-width;
+$container-max-width: tokens.$container-xlarge-max-width;
```

Comment:
I would consider using https://sass-lang.com/documentation/modules/meta/#variable-exists to make this work without "a BC". 

We would check if the variable xlarge exists, if not we would use the original.

And we would make the sizes generator also conditional. 

What do you think?